WIP 3

rendrap opened the pull request:
- add First Visit Section
- fix why-choose-us & firs_visit section
- add corona section inside treatment
- add FAQ page

boneymtom commented:
@rendrap 

1. looks like the stories  link/page is broken or you may be planning to add fix it later
2. FAQ and stories are two different pages 
3.both stories and FAQ to follow google data structures

rendrap commented:
Hi Thomas,

1. Yes, I have not change the link,
I renamed the stories.html to faq.html

2. Should we have separate page for FAQ & stories? Or have only one page
that contains FAQ & stories at the same time?


Regards,

Rendra.

On Thu, Jul 16, 2020, 9:02 AM boneymtom <notifications@github.com> wrote:

> @rendrap <https://github.com/rendrap>
>
>    1. looks like the stories link/page is broken or you may be planning
>    to add fix it later
>    2. FAQ and stories are two different pages
>    3.both stories and FAQ to follow google data structures
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/eltham-family-dental/eltham-family-dental.github.io/pull/14#issuecomment-659111635>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AD5Z5O3VCW6UWVMY6LGWBFLR3ZNTRANCNFSM4O2S2E4A>
> .
>

boneymtom commented:
Hi Rendra ,
 i think we should have a separate FAQ page as the stories pages will need
to be updated frequently


1. Yes, I have not change the link,
I renamed the stories.html to faq.html

2. Should we have separate page for FAQ & stories? Or have only one page
that contains FAQ & stories at the same time?

On Thu, Jul 16, 2020 at 12:09 PM Rendra <notifications@github.com> wrote:

> Hi Thomas,
>
> 1. Yes, I have not change the link,
> I renamed the stories.html to faq.html
>
> 2. Should we have separate page for FAQ & stories? Or have only one page
> that contains FAQ & stories at the same time?
>
>
> Regards,
>
> Rendra.
>
> On Thu, Jul 16, 2020, 9:02 AM boneymtom <notifications@github.com> wrote:
>
> > @rendrap <https://github.com/rendrap>
> >
> > 1. looks like the stories link/page is broken or you may be planning
> > to add fix it later
> > 2. FAQ and stories are two different pages
> > 3.both stories and FAQ to follow google data structures
> >
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/eltham-family-dental/eltham-family-dental.github.io/pull/14#issuecomment-659111635
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AD5Z5O3VCW6UWVMY6LGWBFLR3ZNTRANCNFSM4O2S2E4A
> >
> > .
> >
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/eltham-family-dental/eltham-family-dental.github.io/pull/14#issuecomment-659113691>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALZB5ENSHZKY6XOHRKOLGYLR3ZOONANCNFSM4O2S2E4A>
> .
>


-- 
boneyM

rendrap commented:
Ok Thomas,

I will proceed accordingly.

On Thu, Jul 16, 2020, 9:14 AM boneymtom <notifications@github.com> wrote:

> Hi Rendra ,
> i think we should have a separate FAQ page as the stories pages will need
> to be updated frequently
>
>
> 1. Yes, I have not change the link,
> I renamed the stories.html to faq.html
>
> 2. Should we have separate page for FAQ & stories? Or have only one page
> that contains FAQ & stories at the same time?
>
> On Thu, Jul 16, 2020 at 12:09 PM Rendra <notifications@github.com> wrote:
>
> > Hi Thomas,
> >
> > 1. Yes, I have not change the link,
> > I renamed the stories.html to faq.html
> >
> > 2. Should we have separate page for FAQ & stories? Or have only one page
> > that contains FAQ & stories at the same time?
> >
> >
> > Regards,
> >
> > Rendra.
> >
> > On Thu, Jul 16, 2020, 9:02 AM boneymtom <notifications@github.com>
> wrote:
> >
> > > @rendrap <https://github.com/rendrap>
> > >
> > > 1. looks like the stories link/page is broken or you may be planning
> > > to add fix it later
> > > 2. FAQ and stories are two different pages
> > > 3.both stories and FAQ to follow google data structures
> > >
> > > —
> > > You are receiving this because you were mentioned.
> > > Reply to this email directly, view it on GitHub
> > > <
> >
> https://github.com/eltham-family-dental/eltham-family-dental.github.io/pull/14#issuecomment-659111635
> > >,
> > > or unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/AD5Z5O3VCW6UWVMY6LGWBFLR3ZNTRANCNFSM4O2S2E4A
> > >
> > > .
> > >
> >
> > —
> > You are receiving this because you modified the open/close state.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/eltham-family-dental/eltham-family-dental.github.io/pull/14#issuecomment-659113691
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/ALZB5ENSHZKY6XOHRKOLGYLR3ZOONANCNFSM4O2S2E4A
> >
> > .
> >
>
>
> --
> boneyM
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/eltham-family-dental/eltham-family-dental.github.io/pull/14#issuecomment-659115090>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AD5Z5O7CONG3EP3VAI7DNJTR3ZPBRANCNFSM4O2S2E4A>
> .
>
